afs/media/js/views/components/reports/scenes/name.js

```diff
@@ -110,9 +110,14 @@ define(['underscore', 'knockout', 'arches', 'utils/report','bindings/datatable']
                     }));
                 }
 
-                if(self.dataConfig.exactMatch){
-                    self.exactMatch(self.getNodeValue(params.data(), self.dataConfig.exactMatch));
+                let exactMatchData = params.data()[self.dataConfig.exactMatch];
+                if(exactMatchData) {
+                    if(exactMatchData.length === undefined){
+                        exactMatchData = [exactMatchData]
+                    }
+                    self.exactMatch(exactMatchData.map(x => x["@display_value"]))
```

Comment:
Instead of directly accessing @display_value here, it'd be better to call self.getNodeValue on the node itself - that way it can account for the display of undefined values and such.  so self.getNodeValue(x).  In addition, it's probably better to call self.getRawNodeValue instead of accessing self.dataConfig.exactMatch array style (L 113)

Reply:
updated